src/transformers/models/auto/tokenization_auto.py

```diff
@@ -216,6 +218,10 @@
                 "squeezebert",
                 ("SqueezeBertTokenizer", "SqueezeBertTokenizerFast" if is_tokenizers_available() else None),
             ),
+            (
+                "squeezebert",
+                ("SqueezeBertTokenizer", "SqueezeBertTokenizerFast" if is_tokenizers_available() else None),
+            ),
```

Comment:
```suggestion
```

Reply:
Thank you very much for the catch ! Fixed in 5b9a94b5664dd0501ff5107ca1fbc55b7b885a98